app/lib/handlers/pen.dart

```diff
@@ -342,13 +336,13 @@ class PenHandler extends Handler<PenTool> with ColoredHandler {
 
             elements.clear();
             context.refresh();
-          } else if (points.length > 700) {
-            return;
           }
 
         default:
         // Manage custom shapes here
       }
+    } else if (points.length > 500) {
```

Comment:
i don't understand why else if and not else?
If points < 500 and it doesn't recognize anything it clears it?
I don't think that's really good.

I would just make a guard clause and return if element == null or points > 500 (but I would remove this once we fixed the issue)

Reply:
It's a leftover.  Else enough.  If you can change it as well